src/main/java/Storage.java

```diff
@@ -0,0 +1,77 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+
+public class Storage {
+
+    private String edit;
```

Comment:
Add a Javadoc comment for this since it is not very obvious what it does at first glance. :)